### PR TITLE
increase the logs from containerd version  1.5.10

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

For we have further info in the logs when we are installing and configuring containerd. 

#### Which issue(s) this PR fixes:

Related to # [sc-76762]

#### Special notes for your reviewer:

Check the template: https://github.com/replicatedhq/kURL/pull/4506/files#diff-8a7133184a76cd0383e373a9cff3a1b6c7c5f68233b702bc3ae0e101d2f003ff

The addons installs.sh were updated via its script. 

#### Does this PR introduce a user-facing change?
```release-note
Adds further log information for containerd installs and configuration on versions equals and upper than 1.5.10.
```

### CODE REVIEW

File: addons/containerd/template/base/install.sh [hunk 1]
```diff
@@ -176,6 +202,8 @@ function containerd_configure_proxy() {
 
     echo "Environment=\"HTTP_PROXY=${PROXY_ADDRESS}\" \"HTTPS_PROXY=${PROXY_ADDRESS}\" \"NO_PROXY=${NO_PROXY_ADDRESSES}\"" >> $file
 
+    log "Environment=\"HTTP_PROXY=${PROXY_ADDRESS}\" \"HTTPS_PROXY=${PROXY_ADDRESS}\" \"NO_PROXY=${NO_PROXY_ADDRESSES}\""
```
Comment (camilamacedo86) [inline addons/containerd/template/base/install.sh:205]:
```suggestion
```

File: addons/containerd/template/base/install.sh [hunk 2]
```diff
@@ -162,12 +183,17 @@ function containerd_configure_limitnofile() {
...
     local previous_proxy="$(cat /etc/systemd/system/containerd.service.d/http-proxy.conf 2>/dev/null | grep -io 'https*_proxy=[^\" ]*' | awk 'BEGIN { FS="=" }; { print $2 }')"
     local previous_no_proxy="$(cat /etc/systemd/system/containerd.service.d/http-proxy.conf 2>/dev/null | grep -io 'no_proxy=[^\" ]*' | awk 'BEGIN { FS="=" }; { print $2 }')"
+    log "Previous proxy: ($previous_proxy)"
+    log "Previous no proxy: (previous_no_proxy)"
     if [ "$PROXY_ADDRESS" = "$previous_proxy" ] && [ "$NO_PROXY_ADDRESSES" = "$previous_no_proxy" ]; then
+        log "No changes were found. Proxy configuration still the same"
         return
     fi
 
+    log "Updating proxy configuration with new changes"
```
Comment (camilamacedo86) [inline addons/containerd/template/base/install.sh:196]:
```suggestion
    log "Updating proxy configuration: HTTP_PROXY=${PROXY_ADDRESS} NO_PROXY=${NO_PROXY_ADDRESSES}"
```

File: addons/containerd/template/base/install.sh [hunk 3]
```diff
@@ -193,7 +193,7 @@ function containerd_configure_proxy() {
         return
     fi
 
-    log "Updating proxy configuration with new changes"
+    log "Updating proxy configuration: HTTP_PROXY=${PROXY_ADDRESS} NO_PROXY=${NO_PROXY_ADDRESSES}"
```
Comment (emosbaugh) [inline addons/containerd/template/base/install.sh:196]: I was hoping you could make this change across all files.

Comment (camilamacedo86) [reply addons/containerd/template/base/install.sh:196]: Yes it is done/

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**